### ISSUE TITLE
Update profile size service to use property path input

### ISSUE THREAD

**Issue by balhoff** [phenoscape/phenoscape-kb-services#410]
Something like this for input path `has_character/has_state/describes_phenotype`:

```sparql
SELECT (COUNT(DISTINCT ?phen) AS ?count) 
WHERE {
?item has_character/has_state/describes_phenotype ?phen .
}
```

**Comment by balhoff** [comment]
`?item` will not be a variable here. Profile size is for a particular item that would be input.

**Comment by balhoff** [comment]
Fixed by #432.
